src/mumble/VoiceRecorder.h

```diff
@@ -89,6 +89,9 @@ class VoiceRecorder : public QThread {
 		/// True if multi channel recording is disabled.
 		bool mixDownMode;
 
+		/// True if one of the two transport options is enabled.
```

Comment:
```suggestion
		/// True if a transport recording modes is enabled.
```
Otherwise, the comment is likely to become outdated in the future